src/index.css

```diff
@@ -27,3 +35,44 @@
 #arrowhead {
   fill: var(--color-ui-3);
 }
+
+.json-canvas {
+  --tw-text-opacity: 1;
```

Comment:
Actually are those var needed? I believe they are only for Tailwind

Reply:
Yes, --tw-* is tailwind, and no they should not be needed, I guess they were generated by some "tailwind to css" tool?